packages/design-system/src/interfaces/badge.d.ts

```diff
@@ -0,0 +1,5 @@
+interface IBadgeProps extends React.HTMLAttributes<HTMLSpanElement> {
```

Comment:
omit className, use custom `type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;` because Agent App has old ts version

Reply:
Why is this necessary?

Reply:
because a developer might think it supports `className` property, omit it and he/she will get an error when passing className

Reply:
I decided to add `className` support after all.